src/lib/formats/cp68_dsk.cpp

```diff
@@ -0,0 +1,266 @@
+// license:BSD-3-Clause copyright-holders:Barry Rodewald, 68bit
```

Comment:
Should the copyright holders be updated here?  How much creative content from Barry and 68bit is in this file?

Reply:
This is a good question.  I started with `flex_dsk.cpp` as a base and made necessary modifications for CP/68.  Most of the algorithms/code structure/techniques used are the same as the original with differing details.  Since it seems like this is a derivative work based off the original I'll add myself as an additional copyright holder in the two newly added files as well as a note describing the origin.